src/backend/distributed/shared_library_init.c

```diff
@@ -329,6 +329,12 @@ _PG_init(void)
 						PGC_S_OVERRIDE);
 	}
 
+	/*
+	 * When the options change on a columnar table, we may need to propagate
+	 * the changes to shards.
+	 */
+	ColumnarTableSetOptions_hook = ColumnarTableSetOptionsHook;
```

Comment:
minor, maybe move it up to the other hook assignments

Reply:
Done.